LayoutTests/platform/ios/compositing/tiling/tile-cache-zoomed-expected.txt

```diff
@@ -4,18 +4,18 @@
   (visible rect 0.00, 0.00 800.00 x 600.00)
   (coverage rect 0.00, 0.00 800.00 x 600.00)
   (intersects coverage rect 1)
-  (contentsScale 2.00)
+  (contentsScale 3.20)
```

Comment:
It was passing because the scale was not actually applied at the time layer tree was snapshotted?

Reply:
Indeed. I didn't actually check the history here, but somehow we ended up with iOS-specific expectations that were clearly incorrect.